crates/benchmark-runner/src/guest_programs.rs

```diff
@@ -50,7 +50,7 @@ pub fn stateless_validator_inputs(
         .map(|bw| {
             let mut stdin = Input::new();
             stdin.write(bw.block_and_witness.clone());
-            stdin.write(bw.network);
+            stdin.write(bw.chain_config);
```

Comment:
What we pass now to the guest-program is in `ere-guests/libs/src/chainconfig.rs`.
Basically a list of supported named networks, or an EEST-variant indicating which fork is active since genesis.